test/parallel/test-fs-read-empty-buffer.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+require('../common');
+const fixtures = require('../common/fixtures');
+const assert = require('assert');
+const fs = require('fs');
+const filepath = fixtures.path('testcodes.txt');
+const fd = fs.openSync(filepath, 'r');
+
+const buffer = new Uint8Array();
+
+assert.throws(
+  () => fs.readSync(fd, buffer, 0, 10, 0),
+  {
+    code: 'ERR_INVALID_ARG_VALUE',
```

Comment:
Can you validate the message? Something like

```
message: 'The argument \'buffer\' is empty and cannot be written. ' +
         'Received <Buffer >'` 
```

should work (or whatever the actual error message is)